gnome-image-installer/pages/confirm/Makefile.am

```diff
@@ -0,0 +1,28 @@
+
+noinst_LTLIBRARIES = libgisconfirm.la
+
+AM_CPPFLAGS = \
+	$(INITIAL_SETUP_CFLAGS) $(IMAGE_INSTALLER_CFLAGS) \
+	-DLOCALSTATEDIR="\"$(localstatedir)\"" \
+	-DUIDIR="\"$(uidir)\""
+
+BUILT_SOURCES =
+
+resource_files = $(shell glib-compile-resources --sourcedir=$(srcdir) --generate-dependencies $(srcdir)/confirm.gresource.xml)
+confirm-resources.c: confirm.gresource.xml $(resource_files)
+	$(AM_V_GEN) $(GLIB_COMPILE_RESOURCES) --target=$@ --sourcedir=$(srcdir) --generate-source $<
+confirm-resources.h: confirm.gresource.xml $(resource_files)
+	$(AM_V_GEN) $(GLIB_COMPILE_RESOURCES) --target=$@ --sourcedir=$(srcdir) --generate-header $<
+BUILT_SOURCES += confirm-resources.c confirm-resources.h
+
+libgisconfirm_la_SOURCES =			\
+	gis-confirm-page.c gis-confirm-page.h	\
+	$(BUILT_SOURCES)
```

Comment:
Should `$(BUILT_SOURCES)` be in `nodist_libgisconfirm_la_SOURCES` instead?

Reply:
Almost certainly.